components/pipeline-service/production/stone-prd-m01/resources/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+patches:
+  - path: tekton-results-database-secret-path.yaml
```

Comment:
Nit pick: instead of naming the file `tekton-results-*` I would put them in a `tekton-results` directory.

Reply:
I am just moving out the files to resources. Such customization I would handle it in other PR so that I don't need to mess with rebase a lot.